.circleci/config.yml

```diff
@@ -79,10 +79,16 @@ jobs:
       - checkout
       - run: make install-tools
       - run: make test
+      - run:
+          name: Check Codecov
+          command: |
+            curl -s -o codecov https://codecov.io/bash \
+              && VERSION=$(grep 'VERSION=\".*\"' codecov | cut -d'"' -f2) \
+              && shasum -a 512 -c <(curl -s https://raw.githubusercontent.com/codecov/codecov-bash/${VERSION}/SHA512SUM)
       - run:
           name: Codecov upload
           command: |
-            bash <(curl -s https://codecov.io/bash)
+            bash codecov
```

Comment:
You do not need to specify the shell explicitly as codecov script uses a proper shebang `#!/usr/bin/env bash`

`./codecov` should be enough there